extensions/google-chrome/src/google-chrome-new-tab.tsx

```diff
@@ -0,0 +1,229 @@
+      set _output to ""
+      tell application "Google Chrome"
+        set _window_index to 1
+        repeat with w in windows
+          set _tab_index to 1
+          repeat with t in tabs of w
+            set _title to get title of t
+            set _url to get URL of t
+            set _favicon to ${faviconFormula}
+            set _output to (_output & _title & "${Tab.TAB_CONTENTS_SEPARATOR}" & _url & "${Tab.TAB_CONTENTS_SEPARATOR}" & _favicon & "${Tab.TAB_CONTENTS_SEPARATOR}" & _window_index & "${Tab.TAB_CONTENTS_SEPARATOR}" & _tab_index & "\\n")
+            set _tab_index to _tab_index + 1
+          end repeat
+          set _window_index to _window_index + 1
+          if _window_index > count windows then exit repeat
+        end repeat
+      end tell
+      return _output
+  `);
+
+  return openTabs.split("\n").filter(line => line.length !== 0).map(line => Tab.parse(line));
+}
+
+async function openNewTab(queryText: string | null | undefined): Promise<boolean | string> {
+  popToRoot();
+  closeMainWindow({ clearRootSearch: true });
+  
+  const script = `
+    tell application "Google Chrome"
+      activate
+      tell window 1
+          set newTab to make new tab ` + (queryText ? 'with properties {URL:"http://www.google.com/search?q='+queryText+'"}' : '') + ` 
+      end tell
+    end tell
+  `
+
+  return await runAppleScript(script);
+}
+
+
+
+
+async function setActiveTab(tab: Tab): Promise<void> {
+  await runAppleScript(`
+    tell application "Google Chrome"
+      activate
+      set index of window (${tab.windowsIndex} as number) to (${tab.windowsIndex} as number)
+      set active tab index of window (${tab.windowsIndex} as number) to (${tab.tabIndex} as number)
+    end tell
+  `);
+}
+
+interface State {
+  tabs?: Tab[];
+}
+
+
+export default function Command(): ReactElement {
+    const [searchText, setSearchText] = useState<string>()
+    const { isLoading, error, entries } = useChromeHistorySearch(searchText)
+
+
+    const { useOriginalFavicon } = getPreferenceValues<{ useOriginalFavicon: boolean }>();
+    const [state, setState] = useState<State>({});
+
+    async function getTabs(query: string | null) {
+      let tabs = await getOpenTabs(useOriginalFavicon);
+
+      if(query){
+        tabs = tabs.filter(function (tab) {
+          return (tab.title.toLowerCase().includes(query.toLowerCase()) || tab.urlWithoutScheme().toLowerCase().includes(query.toLowerCase()))
+        })
+      }
+      setState({ tabs: tabs });
+    }
+
+    useEffect(() => { 
+      getTabs(null);
+    }, []);
+
+    if (error) {
+        showToast(ToastStyle.Failure, "An Error Occurred", error.toString())
+    }
+
+    return (
+        <List onSearchTextChange={function (query) { setSearchText(query); getTabs(query);}} isLoading={isLoading} throttle={false}>
+            <List.Section title="New Tab" key="new-tab">
+              <List.Item
+                title={(!searchText ? "Open empty tab" : "Search \""+searchText+"\"")}
```

Comment:
```suggestion
                title={(!searchText ? "Open Empty Tab" : `Search "${searchText}"`)}
```

Corrected naming and added better readability